src/bin/tikv-server.rs

```diff
@@ -97,9 +97,12 @@ fn build_raftkv(matches: &Matches,
     opts.set_max_write_buffer_number(5);
     opts.set_min_write_buffer_number_to_merge(2);
     opts.set_max_background_compactions(3);
-    opts.set_max_bytes_for_level_base(512 * 1024 * 1024);
+    opts.set_max_bytes_for_level_base(64 * 1024 * 1024);
```

Comment:
According to usual practice, the size of L0 should be approximately equal to the size of L1, the max_bytes_for_level_base should be <the size of L0 file> * options.level0_file_num_compaction_trigger. In our case, L0 is ~10M, so make max_bytes_for_level_base 64M is OK.